public-ui/src/app/data-browser/charts/chart-biosex/chart-biosex-react.component.tsx

```diff
@@ -0,0 +1,260 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { environment } from 'environments/environment';
+import * as highCharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import * as ReactDOM from 'react-dom';
+import { BaseReactWrapper } from '../../../data-browser/base-react/base-react.wrapper';
+import { baseOptions, GENDER_STRATUM_MAP } from '../../../data-browser/charts/react-base-chart/base-chart.service';
```

Comment:
Nit: Use the full path instead of relative paths for imports:
```suggestion
import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
import { baseOptions, GENDER_STRATUM_MAP } from 'app/data-browser/charts/react-base-chart/base-chart.service';
```

Reply:
Changed this.